File: python-backend/src/functions/delete_todo.py
from src.utils.package import package
from src.utils.dynamodb_utils import todosTableName, ddb_client

def handler(event, context):
    id = event['pathParameters']['id']

    response = ddb_client.delete_item(
        TableName=todosTableName,
        Key={'todosId': {'S': id}}
    )

    return package(200, response)

File: node-backend/src/functions/deleteTodo.js
import { formatJSONResponse } from '../utils/utils.js'
import { ddbClient, todosTableName } from '../utils/dynamodbUtils.js'

export const handler = async (event) => {
  const id = event.pathParameters.id

  await ddbClient.delete({
    TableName: todosTableName,
    Key: { todosId: id }
  }).promise()

  return formatJSONResponse(200, { id })
}
